### PR TITLE
type-for-the-entries-is-not-a-good-name

### PR DESCRIPTION
**Pull request by myroslavarm**
changed type to kind

### CODE REVIEW

File: src/NECompletion/CompletionModel.class.st [hunk 1]
```diff
@@ -146,9 +146,9 @@ CompletionModel >> sorter: aSorter [
 
 { #category : #accessing }
 CompletionModel >> title [
-	| type |
+	| kind |
 	[
-	type := node receiver propertyAt: #type ifAbsent: [ ^ '' ].
-	^ type name
+	kind := node receiver propertyAt: #type ifAbsent: [ ^ '' ].
```
Comment (jecisc) [inline src/NECompletion/CompletionModel.class.st]: You could also write it

`^ node receiver propertyAt: #type ifPresent: #name ifAbsent: ''`

**Review by MarcusDenker — CHANGES_REQUESTED** [review]
exactly here #type is a good name... the idea was that we use #type when we talk about types, but *not* for the method that returns the symbol for choosing the highlight color.

This is the method #type on the Entries hierarchy, which should be named maybe very explicitly #hightlightSymbol or something like that.

**Comment by myroslavarm** [comment]
@MarcusDenker ok yeah sorry, fixed the method now - should be fine. And also i did update #title according to Cyril's suggestion, too.

**Review by MarcusDenker — APPROVED** [review]
Yes! Looks good.

**Pull request merged**